src/main/java/bio/terra/service/dataset/flight/create/DatasetCreateFlight.java

```diff
@@ -35,16 +31,9 @@ public DatasetCreateFlight(FlightMap inputParameters, Object applicationContext)
         ResourceService resourceService = (ResourceService) appContext.getBean("resourceService");
         BigQueryPdao bigQueryPdao = (BigQueryPdao) appContext.getBean("bigQueryPdao");
         IamProviderInterface iamClient = (IamProviderInterface) appContext.getBean("iamProvider");
-        ApplicationConfiguration appConfig = (ApplicationConfiguration) appContext.getBean("applicationConfiguration");
         ConfigurationService configService = (ConfigurationService) appContext.getBean("configurationService");
         ProfileService profileService = (ProfileService) appContext.getBean("profileService");
 
-        addStep(new VerifyAuthorizationStep(
```

Comment:
How are the permissions on `CREATE_DATASET` validated now?

Reply:
CREATE_DATASET is no longer required. The only check is that the user has the LINK action on the underlying spend profile